recipes/gencove/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2.4.3" %}
+{% set sha256 = "14d16e455142d798f680fa18b1d39377fbb0fea1d69f174bcb1b114038de5286" %}
+
+package:
+  name: gencove
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/gncv/gencove-cli/archive/refs/tags/{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: python
+  number: 0
+  script: '$PYTHON -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv '
+
+requirements:
+  build:
+    - pip
+    - setuptools
+  host:
+    - python >=3.7
+    - pytest-runner
+  run:
+    - python >=3.7
+    - python-dateutil >=2.2.0
+    - six >=1.5
+    - click >=7.0
+    - requests >=2.19.1
+    - boto3 >=1.17.97
+    - progressbar2 ==3.55.0
+    - backoff ==1.11.0
+    - pydantic ==1.9.2
+    - pytest-runner
+
+test:
+  commands:
+    - gencove --help
+
+about:
+  home: https://docs.gencove.com
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: The Gencove command-line interface (CLI) can be used to easily access the API.
```

Comment:
Could you improve this a bit for those who might not know what Gencove is?

Reply:
```suggestion
  summary: Gencove is a high-throughput, cost-effective platform for genome sequencing and analysis. The Gencove command-line interface (CLI) can be used to easily access the API.
```

Reply:
Like this?

Reply:
Maybe 'the API' -> 'the Gencove API'? Otherwise good

Reply:
Fixed it!